src/app/main.c

```diff
@@ -146,6 +162,27 @@ rpmostree_option_context_parse (GOptionContext *context,
     invocation ? invocation->command->flags : RPM_OSTREE_BUILTIN_FLAG_LOCAL_CMD;
   gboolean use_daemon = ((flags & RPM_OSTREE_BUILTIN_FLAG_LOCAL_CMD) == 0);
 
+  if (invocation && invocation->command->description != NULL)
```

Comment:
Oh wait..... you are right.................. I did not realize that -_-, thanks for pointing that out :)